module/VuFind/src/VuFind/Http/CachingDownloader.php

```diff
@@ -90,20 +111,21 @@ public function __construct(HttpService $httpService, CacheManager $cacheManager
      *
      * @return void
      */
-    public function setCacheId($cacheId)
+    public function setCache($cacheId)
```

Comment:
This is still setting the cache ID rather than the cache object. Should we keep the original method name of `setCacheId`? Or alternatively use a name like `resetCache` or `setUpCache` to make it clearer what is happening here. I think it's misleading to make this resemble a setter that mirrors `getCache`, since that's not what it is.